src/config.ts

```diff
@@ -98,26 +98,31 @@ export const refreshConfig = (config: Config, prefsFilePath: string) => {
   config.isLiveServer = getIsLiveServer(webHostUrl, webAppPrefs.siteHost)
   config.emailMode = getEmailOperationMode(serverPrefs.emailTestMode, serverPrefs.testingEmail)
 
-  //Update scheduled jobs from prefs
-  if (serverPrefs.hoursSchedule) {
-    reschedule('action', serverPrefs.hoursSchedule)
-  }
-  if (serverPrefs.actionSchedule) {
-    reschedule('action', serverPrefs.actionSchedule)
-  }
-  if (serverPrefs.backupSchedule) {
-    reschedule('backup', serverPrefs.backupSchedule)
+  // Update locale and timezone if changed
+  const newLocale = serverPrefs.locale ?? Intl.DateTimeFormat().resolvedOptions().locale
+  if (newLocale !== Settings.defaultLocale) {
+    console.log(`Changing locale to: ${newLocale}`)
+    Settings.defaultLocale = newLocale
   }
-  if (serverPrefs.previewDocsCleanupSchedule) {
-    reschedule('cleanup', serverPrefs.previewDocsCleanupSchedule)
+
+  const newTimezone = serverPrefs?.timezone ?? Intl.DateTimeFormat().resolvedOptions().timeZone
+  if (newTimezone !== Settings.defaultZoneName) {
+    console.log(`Changing timezone to: ${newTimezone}`)
+    Settings.defaultZoneName = newTimezone
   }
-  if (serverPrefs.archiveSchedule) {
-    reschedule('archive', serverPrefs.archiveSchedule)
+
+  //Update scheduled jobs from prefs
+  if (config.scheduledJobs) {
+    config.scheduledJobs.reschedule('action', serverPrefs.actionSchedule)
+    config.scheduledJobs.reschedule('backup', serverPrefs.backupSchedule)
+    config.scheduledJobs.reschedule('cleanup', serverPrefs.fileCleanupSchedule)
+    config.scheduledJobs.reschedule('archive', serverPrefs.archiveSchedule)
```

Comment:
Ideally we would compare these to the existing and only update if changed, but it's not trivial to compare them due to the input structure being quite different to how they're stored internally, so not really worth the hassle. There's not really a problem rescheduling them to the same time (if unchanged)